src/app/components/navbar/Navbar.tsx

```diff
@@ -0,0 +1,54 @@
+"use client";
+
+import Link from "next/link";
+import { useState } from "react";
+import { FiMenu, FiX } from "react-icons/fi";
+import NavLinks from "./NavLinks"; 
+import Image from "next/image";
+
+const Navbar = () => {
+  const [menuOpen, setMenuOpen] = useState(false);
+
+  return (
+    <nav className="fixed top-0 z-50 flex items-center justify-between w-full px-6 py-4 bg-white">
+      <div className="flex items-center space-x-2">
+      <Image src="/home/logo.svg" alt="logo" width={120} height={40} priority />
+      </div>
+
+      {/* Navigation Links */}
+      <NavLinks />
+
+      {/* Buttons */}
+      <div className="hidden space-x-6 md:flex">
+        <Link href="/demo" className="text-[var(--gray)] hover:text-[var(--primary)] transition-all px-4 py-2">Request a Demo</Link>

```

Comment:
if you difned variables in config file you can use 
```suggestion
        <Link href="/demo" className="text-gray hover:text-primary transition-all px-4 py-2">Request a Demo</Link>

``` change all the classes like this

Reply:
Done aiye